tests/Permission.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Spatie\Permission\Test;
+
+class Permission extends \Spatie\Permission\Models\Permission
```

Comment:
In my opinion, this is no different from testing with the original model and does not in any way check for possible backward incompatible changes.

Reply:
[#1816](https://github.com/spatie/laravel-permission/issues/1816#issuecomment-903523159) It checks when some field is hidden on custom models(for now, can be used for avoid other break on custom models)
```php
protected $visible = [
      'id', 
      'name',
    ];
```